integrations/express/middleware.ts

```diff
@@ -139,6 +139,11 @@ export function afterMiddleware(keploy: Keploy, req: Request, res: Response) {
     return;
   }
 
+  //to avoid recording tests in the test mode and only recording in the record mode
+  if (process.env.KEPLOY_MODE == "test" && id === undefined) {
```

Comment:
Please use MODE_TEST constant in if check

Reply:
Yes, I will make this change. Actually, in the whole file `express/middleware.ts` this enum( I think) is not used so should replace it everywhere else also or just here?

Reply:
LGTM